README.md

```diff
@@ -54,37 +63,26 @@ Before building you will need OpenCL to be installed, on Ubuntu this can be achi
 > cargo test --all
 ```
 
-## Examples
-
-```
-> cargo build --all --examples --release
-```
+## Benchmarks
 
-Running them
+The main benchmarking tool is called `benchy`.  `benchy` has several subcommands, including `merkleproofs`, `prodbench`, `winning_post` and `window_post`.  You can run them with various configuration options, but some examples are below:
 
 ```
-> ./target/release/examples/merklepor
-> ./target/release/examples/drgporep
-> ./target/release/examples/drgporep-vanilla
-> ./target/release/examples/drgporep-vanilla-disk
-```
+> cargo run --bin benchy -- prodbench
+> cargo run --bin benchy -- merkleproofs --size 2
+> cargo run --bin benchy -- winning-post --size 2
+> cargo run --bin benchy -- window-post --size 2
 
-## Benchmarks
 
-```
-> cargo bench --all
 ```
 
-To benchmark the examples you can use [benchy](src/bin/benchy/main.rs), [stacked](src/bin/benchy/stacked.rs), [prodbench](src/bin/benchy/prodbench.rs), [merkleproofs](src/bin/benchy/merkleproofs.rs], [winning_post](src/bin/benchy/winning_post.rs), or [window_post](src/bin/benchy/window_post.rs).
+There is also a bench called `gpu-cpu-test`:
 
 ```
-# build the script
-> cargo build
-# run the benchmarks
-> ./target/release/benchy
+> cargo run --bin gpu-cpu-test
```

Comment:
```suggestion
> cargo run --release --bin gpu-cpu-test
```

Else it's super slow and not really meaningfull.

Reply:
Updated.  I also expanded the scope of the PR a little.